components/layout.tsx

```diff
@@ -112,7 +112,7 @@ const TopNav: React.FC = () => {
                 style={{ alignContent: "flex-end" }}
                 onClick={closeNav}
               >
-                <NavLink href="/learnbasicsoftestimony">
+                <NavLink href="/learn/basics-of-testimony">
                   Learn About Testimony
                 </NavLink>
                 <NavLink href="/legprocess">
```

Comment:
Could you also rename the legprocess and additionalresources pages as they appear [here](https://docs.google.com/spreadsheets/d/1pIoQtvl62JbFSXAyKwg6T34hd87iUWDs0WswXj9GkUw/edit#gid=0)? Next should be able to handle both the slug and these specific pages in the same folder